pages/faqs.tsx

```diff
@@ -0,0 +1,148 @@
+import { useState, useRef } from "react";
+import { Layout } from "../components/layout";
+import database, { FAQDATA } from "../lib/faq-databases";
+
+type FAQSPROPS = {
+  questionList: FAQSLIST;
+};
+
+type FAQSLIST = {
+  [key: string]: FAQDATA[];
+};
+
+function groupBy<T, U>(data: T[], key: U) {
+  return data.reduce((acc: any, currentValue: any) => {
+    let groupKey = currentValue[key];
+    if (!acc[groupKey]) {
+      acc[groupKey] = [];
+    }
+    acc[groupKey].push(currentValue);
+    return acc;
+  }, {});
+}
+
+export default function Faqs(props: FAQSPROPS) {
+  const { questionList } = props;
+  const inputRef = useRef<HTMLInputElement>(null);
+  const [questionData, setQuestionData] = useState(questionList);
+
+  const onSearchQuestion = () => {
+    if (!inputRef.current!.value) {
+      return setQuestionData(questionList);
+    }
+
+    const searchResult: FAQSLIST = {};
+    Object.keys(questionList).map((category: string) => {
+      const questionCategory = questionList[category];
+      for (let i = 0; i < questionCategory.length; i++) {
+        const pertanyaan = questionCategory[i].pertanyaan.toLowerCase();
+        const keyword = inputRef.current!.value.toLowerCase();
+        if (pertanyaan.includes(keyword)) {
+          if (!searchResult[category]) {
+            searchResult[category] = [];
+          }
+          searchResult[category].push(questionCategory[i]);
+        }
+      }
+    });
+    setQuestionData(searchResult);
+  };
+
+  return (
+    <Layout>
+      <header>
+        <h1>
+          {/* eslint-disable-next-line @next/next/no-img-element */}
+          <img
+            src="https://firebase-kanvas.imgix.net/warga_bantu_warga/hero_banner.png?auto=format,compress,enhance&fm=pjpg&cs=tinysrgb&fit=scale"
+            alt="Warga Bantu Warga"
+            height="291"
+            width="650"
+          />
+        </h1>
+      </header>
+      <div className="px-2">
+        <div className="max-w-7xl mx-auto py-6">
+          <h2 className="text-3xl font-extrabold text-gray-900">
+            Frequently asked questions
```

Comment:
```suggestion
            Pertanyaan yang sering ditanyakan
```

Please change it into Bahasa Indonesia instead.